global.json

```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "2.1.301"
+    "version": "2.1.403"
```

Comment:
[`2.2.100-preview3`](https://www.microsoft.com/net/download/dotnet-core/2.2)?

Reply:
Honestly I'd prefer to bump to 2.2 in another PR. We did light smoking of 2.1 and 15.9 already.

Reply:
I'm expecting at least wxs changes and maybe other issues, so it would be nice to get this in while it's green.

Reply:
Nevermind, I tested locally and it seems fine. Bumped.